akka-actor/src/main/resources/reference.conf

```diff
@@ -438,6 +441,18 @@ akka {
       mailbox-requirement = ""
     }
 
+    default-blocking-io-dispatcher {
+      type = "Dispatcher"
+      executor = "thread-pool-executor"
+      throughput = 1
+
+      thread-pool-executor {
+        core-pool-size-min = 2
+        core-pool-size-factor = 2.0
+        core-pool-size-max = 16
```

Comment:
amount of blocking threads are typically not limited by number of cores, so it makes more sense to just use a fixed pool

```
  thread-pool-executor {
    fixed-pool-size = 16
  }
```

This is probably taken from akka-streams and I think we should change there also at the same time.
It would maybe have been best to use the same pool for akka-actor and akka-stream, but I'm afraid too many are already relying on configuring the `akka.stream.default-blocking-io-dispatcher` so changing akka-stream to use this one is too dangerous.